ApiBundle/Tests/Transformer/ContentAttributeTransformerTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Tests\Transformer;
+
+use OpenOrchestra\ApiBundle\Transformer\ContentAttributeTransformer;
+use Phake;
+
+/**
+ * Class ContentAttributeTransformerTest
+ */
+class ContentAttributeTransformerTest extends \PHPUnit_Framework_TestCase
+{
+    protected $contentAttribute;
+    protected $transformer;
+
+    /**
+     * set Up
+     */
+    public function setUp()
+    {
+        $this->contentAttribute = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentAttributeInterface');
+        $this->transformer = new ContentAttributeTransformer();
+    }
+
+    /**
+     * test transform
+     */
+    public function testTransform()
+    {
+        $facade = $this->transformer->transform($this->contentAttribute);
+
+        Phake::verify($this->contentAttribute, Phake::times(1))->getName();
```

Comment:
You should test that the facade return contains the data from the contentAttribute